docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerBuildVariableContributor.java

```diff
@@ -31,9 +31,11 @@ public void buildVariablesFor(AbstractBuild build, Map<String, String> variables
             variables.put("JENKINS_CLOUD_ID", dockerComputer.getCloudId());
             try {
                 //replace http:// and https:// from docker-java to tcp://
-                final String dockerHost = new URIBuilder(dockerComputer.getCloud().serverUrl)
-                        .setScheme("tcp")
-                        .toString();
+                final URIBuilder uriBuilder = new URIBuilder(dockerComputer.getCloud().serverUrl);
+                if (uriBuilder.getScheme() != "unix") {
```

Comment:
Equals()

Worry not though, I already changed it when testing locally and will apply
when next sitting at a pc.

On Thursday, 29 October 2015, Eberhard Beilharz <notifications@github.com>
wrote:

> In
> docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerBuildVariableContributor.java
> <https://github.com/jenkinsci/docker-plugin/pull/344#discussion_r43353495>
> :
>
> > @@ -31,9 +31,11 @@ public void buildVariablesFor(AbstractBuild build, Map<String, String> variables
> >              variables.put("JENKINS_CLOUD_ID", dockerComputer.getCloudId());
> >              try {
> >                  //replace http:// and https:// from docker-java to tcp://
> > -                final String dockerHost = new URIBuilder(dockerComputer.getCloud().serverUrl)
> > -                        .setScheme("tcp")
> > -                        .toString();
> > +                final URIBuilder uriBuilder = new URIBuilder(dockerComputer.getCloud().serverUrl);
> > +                if (uriBuilder.getScheme() != "unix") {
>
> I'm new to both Java and Groovy - so tell me, how would it properly look
> in Java then?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/docker-plugin/pull/344/files#r43353495>.
>